explorer/explorerroutes.go

```diff
@@ -140,30 +143,90 @@ func (exp *explorerUI) DisapprovedBlocks(w http.ResponseWriter, r *http.Request)
 	io.WriteString(w, str)
 }
 
+// show only regular tx in block.Transactions, exclude coinbase (reward) transactions
+// for use in NextHome handler and websocket response to getmempooltxs event
+func filterRegularTx(txs []*TrimmedTxInfo) []*TrimmedTxInfo {
+	transactions := make([]*TrimmedTxInfo, 0)
+	for _, tx := range txs {
+		if !tx.Coinbase {
+			transactions = append(transactions, tx)
+		}
+	}
+	return transactions
+}
+
 // NextHome is the page handler for the "/nexthome" path.
 func (exp *explorerUI) NextHome(w http.ResponseWriter, r *http.Request) {
+	// Get top N blocks and trim each block to have just the fields required for this page.
 	height := exp.blockData.GetHeight()
+	blocks := exp.blockData.GetExplorerFullBlocks(height, height-homePageBlocksMaxCount)
+
+	// trim unwanted data in each block
+	trimmedBlocks := make([]*TrimmedBlockInfo, 0, len(blocks))
+	for index := range blocks {
+		block := blocks[index]
+
+		trimmedBlock := &TrimmedBlockInfo{
+			Time:         block.BlockTime,
+			Height:       block.Height,
+			Total:        block.TotalSent,
+			Fees:         block.MiningFee,
+			Subsidy:      block.Subsidy,
+			Votes:        block.Votes,
+			Tickets:      block.Tickets,
+			Revocations:  block.Revs,
+			Transactions: filterRegularTx(block.Tx),
+		}
 
-	blocks := exp.blockData.GetExplorerFullBlocks(height, height-11)
+		trimmedBlocks = append(trimmedBlocks, trimmedBlock)
+	}
 
 	exp.pageData.RLock()
 	exp.MempoolData.RLock()
 
+	mempoolVotes := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Votes)
+	mempoolTickets := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Tickets)
+	mempoolRevs := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Revocations)
+	mempoolTxs := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Transactions)
+
```

Comment:
If you grab the `exp.MempoolData.TotalOut` and `.MempoolData.LastBlockTime` values here, you can unlock `exp.MempoolData` much earlier.